src/file/file.service.spec.ts

```diff
@@ -49,11 +50,14 @@ const ExceedQuotaSize: number = 11 * GB;
 describe('File Logic', () => {
 
   before(async () => {
-    // Remove files from DB
+    await mongoose.connection.db.dropDatabase();
     const collections = ['files', 'uploads'];
     for (const i in collections) {
       mongoose.connection.db.createCollection(collections[i], (err) => { });
     }
```

Comment:
What are these lines for? and why is quota not there if they are necessary?

Reply:
You are the one who wrote that.